src/components/molecules/BookCard/BookCard.tsx

```diff
@@ -91,36 +91,113 @@ export type Book = {
   readTime: string;
   readersCount: string;
   status: string;
+  category: string;
+  tags:Array<string>
 };
 
 export interface BookCardComponentProps {
   book: Book;
   typeOfCard?: string;
-  onFinishedClick: (arg: any) => void;
+  changeBookStatus: (arg: Book) => void;
   bookObject: Array<Book>;
+  isCategoryTab?: boolean;
+}
+
+const buttonPropTypes = {
```

Comment:
Refactor this component, molecules should not have this. Try to have enums declared to store the defaults